website/react-magma-docs/src/pages/api/stepper.mdx

```diff
@@ -0,0 +1,592 @@
+        {currentStep === 1 && <div>Step Content Two</div>}
+        {currentStep === 2 && <div>Step Content Three</div>}
+        {currentStep === 3 && <div>Step Content Four</div>}
+        {currentStep === 4 && <div>Steps Completed</div>}
+      </Container>
+
+      <Container style={{ padding: '20px 0' }}>
+        <Button style={{ marginRight: '4px' }} onClick={handleOnPrevious}>
+          Previous
+        </Button>
+        <Button onClick={currentStep >= 4 ? handleFinish : handleOnNext}>
+          {currentStep >= 4 ? 'Finish' : 'Next'}
+        </Button>
+      </Container>
+    </>
+  );
+}
+```
+
+## Hide Labels
+
+To hide all labels, `hideLabels` toggles off both the `label` and `secondaryLabel` while retaining the proper accessibility guidelines for screenreaders.
+
+```typescript
+import React from 'react';
+import { Container, Stepper, Step } from 'react-magma-dom';
+
+export function Example() {
+  const [currentStep, setCurrentStep] = React.useState(0);
+
+  const handleOnNext = () => {
+    if (currentStep !== 4) {
+      setCurrentStep(currentStep + 1);
+    }
+  };
+  const handleOnPrevious = () => {
+    if (currentStep !== 0) {
+      setCurrentStep(currentStep - 1);
+    }
+  };
+  const handleFinish = () => {
+    if (currentStep === 4) {
+      setCurrentStep(0);
+    }
+  };
+
+  return (
+    <>
+      <Stepper hideLabels currentStep={currentStep}>
+        <Step label="First Step" secondaryLabel="Description One" />
+        <Step label="Second Step" secondaryLabel="Description Two" />
+        <Step label="Third Step" secondaryLabel="Description Three" />
+        <Step label="Fourth Step" secondaryLabel="Description Four" />
+      </Stepper>
+
+      <Container
+        style={{
+          background: '#F5F5F5',
+          borderRadius: '6px',
+          margin: '20px 0 0',
+          padding: '20px',
+        }}
+      >
+        {currentStep === 0 && <div>Step Content One</div>}
+        {currentStep === 1 && <div>Step Content Two</div>}
+        {currentStep === 2 && <div>Step Content Three</div>}
+        {currentStep === 3 && <div>Step Content Four</div>}
+        {currentStep === 4 && <div>Steps Completed</div>}
+      </Container>
+
+      <Container style={{ padding: '20px 0' }}>
+        <Button style={{ marginRight: '4px' }} onClick={handleOnPrevious}>
+          Previous
+        </Button>
+        <Button onClick={currentStep >= 4 ? handleFinish : handleOnNext}>
+          {currentStep >= 4 ? 'Finish' : 'Next'}
+        </Button>
+      </Container>
+    </>
+  );
+}
+```
+
+## Breakpoints
+
+For responsive layouts, the `breakpoint` prop used along with `breakpointStyle` hides certain elements of the `Stepper` at a specified screen width. `breakpoint` takes a number for the screen width and `breakpointStyle` hides either labels or shows just the summary view. By default, step labels will display if no responsive props are used.
+
+#### Breakpoint - Hide Labels
```

Comment:
This example falls outside the bounds
![image](https://github.com/cengage/react-magma/assets/91160746/32919873-b691-4a2b-8c5e-c8e1320fa8a6)